lib/active_merchant/billing/gateways/secure_trading.rb

```diff
@@ -23,7 +23,16 @@ class SecureTradingGateway < Gateway
         '31004' => STANDARD_ERROR_CODE[:expired_card],
         '31009' => STANDARD_ERROR_CODE[:processing_error],
         '70000' => STANDARD_ERROR_CODE[:card_declined],
-      }
+      }.freeze
+
+      CARD_BRAND_MAP = {
+        'visa'              => 'VISA',
+        'master'            => 'MASTERCARD',
+        'maestro'           => 'MAESTRO',
+        'discover'          => 'DISCOVER',
+        'jcb'               => 'JCB',
+        'american_express'  => 'AMEX',
+      }.freeze
```

Comment:
Only mastercard and amex are needed in this map with the default value you've chosen.